backend/src/main/kotlin/no/uib/echo/routes/UserRoutes.kt

```diff
@@ -268,8 +260,8 @@ fun Route.getAllUsers() {
         }
 
         val users = transaction {
-            User.select { User.email like "%@student.uib.no" or (User.email like "%@uib.no") }
-                .map { it ->
```

Comment:
Denne ble brukt for å filtrere vekk alle legacy-brukere (aka alle unike mailer som noen sinne har meldt seg på før i fjor en gang), siden de har hovedmailer som er random domener. Blir veldig mange flere brukere om de tas med.

Reply:
Jo, sant, det var derfor den var der. Ok, da skal jeg la den være.